<launch>
	<!--
	<param name="/cv_proj/input_file" type="string" value="/home/igor/pcds/cluttered/3_objs_ardrone_outdoor.pcd" />
	
	<param name="/cv_proj/template_file" type="string" value="/home/igor/pcds/templates/indoor_template.pcd" />
	-->

	<!--
	<rosparam param="/cv_proj/input_file">"/home/igor/pcds/cluttered/3_objs_ardrone_indoor.pcd"</rosparam>
	<rosparam param="/cv_proj/template_file">"/home/igor/pcds/templates/output_cv_proj/indoor_full_01.pcd"</rosparam>
	-->
	<rosparam param="/cv_filter_batch/input_dir">"/home/igor/pcds/pcd_grabber_batch"</rosparam>
	<!--
	<rosparam param="/cv_proj/input_file">"/home/igor/pcds/assembly_objs/ardrone_03_outdoor.pcd"</rosparam>
	<rosparam param="/cv_proj/template_file">"/home/igor/pcds/templates/output_cv_proj/naked_full_01.pcd"</rosparam>
	-->

	<node pkg="cv_filter_batch" type="cv_filter_batch" name="cv_filter_batch" output="screen"/>
	
</launch>

<!--
<param name="/cv_proj/output_file" type="string" value="/home/igor/pcds/cv_proj_out/out_cluttered_indoor_01.pcd" />
	<param name="/cv_proj/out_rgb" type="string" value="/home/igor/pcds/cv_proj_out/out_result_05.jpg" />
-->
